common/src/main/java/com/wynntils/overlays/minimap/MinimapOverlay.java

```diff
@@ -42,6 +43,7 @@
 import java.util.stream.Stream;
 import net.minecraft.client.gui.Font;
 import net.minecraft.client.renderer.MultiBufferSource;
+import org.joml.Math;
```

Comment:
No, don't do this. We have MathUtils.

Reply:
I thought Spotless should have removed the unused import..?

Have you disabled running github actions on your fork?

Reply:
Or maybe you unchecked the box to allow maintainers to edit the branch..?

Somehow our Spotless checks are failing to update this branch, anyway...

Reply:
i also ran spotless

Reply:
Then why is this import still here..?

Reply:
i think i forgot to push

Reply:
nvm branch is up to date. the change should be online but I can't see the changes on github only the push msg

Reply:
now i can see the changes - i had the refresh the branch. Maybe this doesn't get updated because it is the only change I the file and spotless can't change an edited file to an unedited one.

Reply:
the change is there but you still didnt remove the import?

Reply:
it didn't remove the import because it was used. Before the class was using java.lang.Math without import and because I imported joml Math it switch to that one and didn't switch back.